templates/zerver/help/configure-email-notifications.md

```diff
@@ -0,0 +1,21 @@
+# Configure email notifications
+
+Zulip allows you to configure your email notification preferences. You have a choice to receive or not receive email notification when you're offline.
+
+You can easily change your email notification settings by following these steps.
+
+1. Click the cog (![cog](/static/images/help/cog.png)) in the upper right   corner of the right sidebar.
+
+2. Select **Settings** from the dropdown menu that appears.<br>
+
+    ![Settings](/static/images/help/cog_dropdown_settings.png)
+
```

Comment:
In many docs we add an additional step, describing what clicking an option will do - e.g. [here](https://github.com/zulip/zulip/blob/master/templates/zerver/help/deactivate-or-reactivate-a-user.md). Could you add it in your doc?